packages/components/PrintStatementPanel/PrintStatementPanel.module.css

```diff
@@ -0,0 +1,118 @@
+.MessagePrefixBadgeUnicorn,
+.MessagePrefixBadgeSolidColor {
+  flex-shrink: 0;
+  align-self: center;
+}
+.MessagePrefixBadgeUnicorn {
+  width: 1.25rem;
+  height: 1.25rem;
+
+  /* HACK to horizontally align text; this icon needs to match the MessageIcon width. */
+  margin: 0 calc((18px - 1.25rem) / 2);
+}
+.MessagePrefixBadgeSolidColor {
+  border-radius: 100%;
+  width: 0.5rem;
+  height: 0.5rem;
+
+  /* HACK to horizontally align text; this icon needs to match the MessageIcon width. */
+  margin: 0 calc((18px - 0.5rem) / 2);
+}
+
+.PrefixBadgeButton {
+  position: relative;
+  height: 1.25rem;
+}
+
+.ColorBadgeButton {
+  border-radius: 100%;
+  height: 1.25rem;
+  width: 1.25rem;
+}
+
+.ColorBadgeButtonEmpty {
+  background-image: url(~devtools/client/debugger/images/sources/prefix-picker.svg) !important;
+}
+
+.PopOutContainer {
+  /* position: absolute; */
+  right: -0.25rem;
+  top: -0.25rem;
+  display: flex;
+  align-items: center;
+  gap: 0.25rem;
+  padding: 0.25rem;
+  border-radius: 1rem;
+  box-shadow: 0px 1px 2px 0px #00000040;
+}
+:root:global(.theme-light) .PopOutContainer {
+  background: #fff;
+}
+:root:global(.theme-dark) .PopOutContainer {
+  background: #474c52;
+}
+
+.PickerItem {
+  cursor: pointer;
+}
+
+.ColorBadge,
+.UnicornBadge {
+  width: 1.25rem;
+  height: 1.25rem;
+  flex: 0 0 1.25rem;
+}
+.ColorBadge {
+  border-radius: 100%;
+}
+.UnicornBadge {
+  background-image: url(~devtools/client/debugger/images/sources/unicorn-pixel.svg) !important;
+  background-size: auto 100%;
+}
+
+/* Background fill colors */
+
+:root:global(.theme-light) .unicorn {
+  background: #ff6ddf;
```

Comment:
What do you think about standardizing here on either hex or rgb?

Reply:
I'd love that! We need to do a pass on our theming and cleanup all of the variable names, I'll make sure to account for values as well. Would be nice to try the new color models available that are more human configurable like LCH.